src/components/@core/layout/page-heading.tsx

```diff
@@ -0,0 +1,32 @@
+import { Heading, Stack } from "@chakra-ui/react";
+import React, { ReactNode } from "react";
+
+interface PageHeadingProps {
+  children?: ReactNode;
+  mb?: number;
+  actions?: ReactNode;
+  [x: string]: any;
+}
+
+export default function PageHeading({
+  children,
+  className,
+  mb = 4,
```

Comment:
can you please rename this prop to something more descriptive?like calling it marginBottom?